modules/MouseModels.py

```diff
@@ -728,6 +716,36 @@ def generate_phenodigm_evidence_strings(self, upper_limit=0):
                                             #self._logger.error("hs_symbol in disease_gene_locus[disease_id]: {0}".format(not disease_term_uris == None and disease_id in self.disease_gene_locus and hgnc_gene_id in self.disease_gene_locus[disease_id]))
                                             #self._logger.error("marker_symbol in disease_gene_locus[disease_id][hgnc_gene_id]): {0}".format(disease_term_uris is not None and disease_id in self.disease_gene_locus and marker_symbol in self.disease_gene_locus[disease_id][hgnc_gene_id]))
 
+    def convert_evidence(self, evidenceString):
+        '''
+        This function maps the old evidence structure into the new json schema.
+
+        Args:
+            evidenceString (dict): old PhenoDigm evidence string
+        Returns:
+            dict: dictionary for the new evidence if all fields could be successfully mapped. 
+        '''
+
+        try:
+            return collections.OrderedDict({
```

Comment:
Is it possible that in this function `data` has to be replaced by `evidenceString`?

Reply:
Absolutely. Fixed.